giraffe/webpack.common.config.js

```diff
@@ -57,31 +57,14 @@ module.exports = {
       {
         test: /\.css$/,
         exclude: /leaflet\.css/,
-        use: [
-          {loader: 'style-loader'},
-          {loader: 'css-loader'},
-        ],
+        use: [{loader: 'style-loader'}, {loader: 'css-loader'}],
```

Comment:
```suggestion
        use: [{loader: 'style-loader'}, 'css-loader'],
```

I think we can go even deeper here, and make these forms identical.